components/suggestion/command_provider.jsx

```diff
@@ -115,7 +126,14 @@ export default class CommandProvider extends Provider {
 
     handleWebapp(pretext, resultCallback) {
         const command = pretext.toLowerCase();
-        Client4.getCommandAutocompleteSuggestionsList(command, getCurrentTeamId(store.getState())).then(
+        const teamId = getCurrentTeamId(store.getState());
+
+        const args = {
+            channel_id: this.channelId,
+            ...(this.rootId && {root_id: this.rootId, parent_id: this.rootId}),
```

Comment:
This works? Maybe just make a conditional?

Reply:
It does! Isn't it awesome? At first I was adding it in a conditional, but I was like "there's got to be a better way to do this." I found it here: https://stackoverflow.com/a/40560953/1218899

Reply:
Never seen that before. I guess it's fine 😄